Library/Homebrew/cmd/bottle.rb

```diff
@@ -57,15 +57,8 @@ def print_filename(string, filename)
       # skip document file.
       next if Metafiles::EXTENSIONS.include? file.extname
 
-      # Check dynamic library linkage. Importantly, do not run otool on static
-      # libraries, which will falsely report "linkage" to themselves.
-      if file.mach_o_executable? || file.dylib? || file.mach_o_bundle?
-        linked_libraries = file.dynamically_linked_libraries
-        linked_libraries = linked_libraries.select { |lib| lib.include? string }
-        result ||= linked_libraries.any?
-      else
-        linked_libraries = []
-      end
+      linked_libraries = Keg.file_linked_libraries(file)
```

Comment:
This fails to pass `string` to the method (and the method lacks a `string` argument).